src/App.js

```diff
@@ -21,21 +21,23 @@ function App() {
     <ThemeProvider theme={theme}>
     <Container>
       <BrowserRouter>
-      <Nav themeToggle={toggleTheme} />
-      <Link to="/">Home</Link>
-      <SubNav />
+      <NavigationList themeToggle={toggleTheme} />
       <Routes>
-      <Route exact path="/" element="">Home
-      </Route>
+      <Route exact path="/" element="">Home</Route>
       <Route path="/Portfolio" element="">Portfolio</Route>
-      <Route path="/coins">Coins</Route>
+      {/* <Route path="/coins">Coins</Route> */}
       <Route path="/coin/:coinid" element={<Coin />}></Route>
-    </Routes>
-      <Charts />
-    <div className="CoinTab">
-        <CoinsList />
-      </div>
+    </Routes> 
       </BrowserRouter>
+      <div className="OrganizeCharts">
+        <div className="ChartWrapper">
+        <VolumeChart></VolumeChart>
+        </div>
+        <div className="ChartWrapper">
+        <LineChart></LineChart>
+        </div>
+      </div>
+      <CoinsList></CoinsList>
```

Comment:
do you need this?

Reply:
Yes, those are the rendered components which make up the chart and the coinlist